lib/Controller/DocumentController.php

```diff
@@ -805,12 +805,21 @@ public function wopiCheckFileInfo($documentId) {
 			return new JSONResponse([], Http::STATUS_NOT_FOUND);
 		}
 
+		// make sure file can be read when checking file info
 		$file = $this->getFileHandle($fileId, $res['owner'], $res['editor']);
 		if (!$file) {
 			$this->logger->warning('wopiCheckFileInfo(): Could not retrieve file', ['app' => $this->appName]);
 			return new JSONResponse([], Http::STATUS_NOT_FOUND);
 		}
 
+		// trigger read operation while checking file info for user
+		// after acquiring the token
+		$fileHandle = $file->fopen('rb');
+		if (!$fileHandle) {
```

Comment:
https://drone.owncloud.com/owncloud/richdocuments/1133/4/4
"818    Negated boolean expression is always false."
`phpstan` thinks that the result of `$file->fopen('rb');` can never be false.